c-pallets/file-bank/src/lib.rs

```diff
@@ -424,12 +432,12 @@ pub mod pallet {
 		/// - `file_hash`: The beneficiary related to signer account.
 		/// - `file_size`: File size calculated by consensus.
 		/// - `slice_info`: List of file slice information.
-		/// - `user`: The first user to upload files.
+		/// - `user`: The first user to upload files. // Does it mean origin? I don't see the parameter
 		#[transactional]
-		#[pallet::weight(<T as pallet::Config>::WeightInfo::upload())]
+		#[pallet::weight(<T as pallet::Config>::WeightInfo::upload())] // should depend on slice_info value
```

Comment:
Yes, you are right. I should calculate weight according to slice.
I modified the benchmark code here:
https://github.com/CESSProject/cess/pull/88/files#diff-127d5bc8d5ced709e3871ca77106ae0b18c87710ea3eae9e16437a826d0d603bR267-R296